### PR TITLE
fix implementation issue for instanceNormalization op

### PR DESCRIPTION
**Pull request by BruceDai**
Sorry to introduce this issue by referring to implementation of **batchNormalization** with missing updating **sub** and **add** computing to totally align with pseudo-code of **instanceNormalization** op.

@fdwr @huningxin PTAL, thanks

### CODE REVIEW

File: src/instance_normalization.js [hunk 1]
```diff
@@ -31,10 +31,10 @@ export function instanceNormalization(
   const shape = [1, null, 1, 1];
   const reduceOptions = {axes: [2, 3], keepDimensions: true};
   const mean = reduceMean(input, reduceOptions);
-  output = sub(input, reshape(mean, shape));
+  output = sub(input, mean);
   const variance = reduceMean(pow(output, new Scalar(2)), reduceOptions);
```
Comment (huningxin) [inline src/instance_normalization.js:35]: Why not match the pseudo code?
```js
const variance = reduceMean(pow(sub(input, mean), new Scalar(2)), reduceOptions);
```

File: test/softplus_test.js [hunk 2]
```diff
@@ -11,7 +11,7 @@ describe('test softplus', function() {
     utils.checkValue(outputTensor, expected);
   }
 
-  it.only('softplus default', function() {
+  it('softplus default', function() {
```
Comment (huningxin) [inline test/softplus_test.js:14]: Is this change part of instanceNormalization fix?

File: src/instance_normalization.js (hunk 1 above)
Comment (BruceDai) [reply src/instance_normalization.js:35]: Last I referred to implementation of **batchNormalization**, missed modified **sub** and **add** computing of this implementation. Now it matches, after this fixing.

File: test/softplus_test.js (hunk 2 above)
Comment (BruceDai) [reply test/softplus_test.js:14]: Yes. Last I used `it.only()` for debugging **softplus** tests. 
Do you suggest adding another commit or pr for clearly fixing tests, thanks.

**Review by BruceDai — COMMENTED** [review]
@huningxin  Thanks for your reviewing.

File: src/instance_normalization.js (hunk 1 above)
Comment (huningxin) [reply src/instance_normalization.js:35]: I mean match it exactly by combining line 34 into line 35.

File: test/softplus_test.js (hunk 2 above)
Comment (huningxin) [reply test/softplus_test.js:14]: adding another commit sgtm. Thanks.

**Comment by BruceDai** [comment]
Done to address your comments, please take another look, thanks @huningxin .

**Review by huningxin — APPROVED** [review]
LGTM, thanks @BruceDai !

**Review by fdwr — APPROVED** [review]

**Pull request merged**